src/assets/i18n/en-us.json

```diff
@@ -52,14 +52,15 @@
   "logout": "Log Out",
   "tags": "Tags",
   "all": "All",
-  "viewAll": "View All",
+  "viewAll": "open more...",
```

Comment:
Move the property to `"message.viewAll"` for consistency. The properties in root is for title which is capitalized to its key.